src/server/ua_subscription_datachange.c

```diff
@@ -12,6 +12,7 @@
 #include "ua_server_internal.h"
 #include "ua_subscription.h"
 #include "ua_types_encoding_binary.h"
+#include "math.h"
```

Comment:
Do we need this include? I don't see a specific math operation...

Reply:
This only applies to the other features so it will be removed.